examples/offscreen/simple_osmesa.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# vispy: testskip
+# Copyright (c) 2015, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+"""
+This is a simple osmesa example that produce an image of a cube
+
+If you have both osmesa and normal (X) OpenGL installed, execute with
+something like the following to pickup the OSMesa libraries :
```

Comment:
`libraries::`, plus newline between this and the literal below it, will make it render properly in ReStructuredText